WooCommerce/Classes/Tools/Zendesk/ZendeskManager.swift

```diff
@@ -22,6 +22,10 @@ protocol ZendeskManagerProtocol {
     /// Displays the Zendesk New Request view from the given controller, for users to submit new tickets.
     ///
     func showNewRequestIfPossible(from controller: UIViewController, with sourceTag: String?)
+
+    /// Displays a Zendesk New Request view from the given controller, tagged to show in the WCPay queues, for users to submit new tickets.
+    ///
+    func showNewWCPayRequestIfPossible(from controller: UIViewController, with sourceTag: String?)
```

Comment:
Do we anticipate having a separate request for Stripe Terminal on Stripe merchants? I would assume not, so let's call this `showNewPaymentsRequestIfPossible` or something similar instead of mentioning WCPay specifically.

Reply:
This is an internal API, that is exposed to submit requests that go into a specific zendesk queue, associated to WCPay, and tagged per the HE's request, so it makes sense to be explicit.